nvim/.config/nvim/lua/user/lsp/lsp-installer.lua

```diff
@@ -1,33 +1,42 @@
 local utils = require('user.utils')
 local lsp_installer = utils.safe_require('nvim-lsp-installer')
+local lspconfig = utils.safe_require('lspconfig')
+local lsp_handlers = require('user.lsp.handlers')
 
--- Register a handler that will be called for all installed servers.
--- Alternatively, you may also register handlers on specific server instances instead.
-local tbl_extend = vim.tbl_deep_extend
+lsp_installer.setup({
+  ui = {
+    icons = {
+       server_installed = "✓",
+       server_pending = "➜",
+       server_uninstalled = "✗"
+    }
+  }
+})
 
-lsp_installer.on_server_ready(function(server)
```

Comment:
`on_server_ready` is deprecated and it's recommended to configure each server separately